### PR TITLE
Update guide: Kubernetes monitoring with Netdata: Overview and visualizations

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR dramatically updates the existing k8s monitoring guide to include the recently-released visualizations in Netdata Cloud.

This PR also sneaks in a few small tweaks to the pair of anomaly detection guides to remove mention of a third part.

##### Component Name

area/docs

### CODE REVIEW

**Comment by joelhans** [comment]
[Update: Monitor a Kubernetes cluster with Netdata](https://trello.com/c/LCqFdsw9/29-update-monitor-a-kubernetes-cluster-with-netdata)

**Review by ilyam8 — DISMISSED** [review]

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
 <!--
-title: "Monitor a Kubernetes (k8s) cluster with Netdata"
-description: "Use Netdata's helmchart, service discovery plugin, and Kubelet/kube-proxy collectors for real-time visibility into your Kubernetes cluster."
+title: "Kubernetes monitoring with Netdata: Overview and visualizations"
+description: "TK"
```
Comment (ilyam8) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:3]: out of curiosity, whats TK? 🤔

Comment (joelhans) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:3]: It's publishing world-speak for "to come," which also means I forgot to add this in...

**Review by ilyam8 — DISMISSED** [review]

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+author_title: "Editorial Director, Technical & Educational Resources"
+author_img: "/img/authors/joel-hans.jpg"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/guides/monitor/kubernetes-k8s-netdata.md
 -->
 
-# Monitor a Kubernetes cluster with Netdata
+# Kubernetes monitoring with Netdata: Overview and visualizations
+
+At Netdata, we've built Kubernetes monitoring tools that add visibility without complexity while also helping you
+actively troubleshoot anomalies or outages. This guide walks you through each of the visualizations and offers best
```
Comment (OdysLam) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:14]: Can we argue that we actively help the user? Active help, in my mind, would be something like giving tips/insights to the user based on the underlying metric.

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+practices on how to use them to help you start Kubernetes monitoring in a matter of minutes, not hours or days.
+
+We already have a few complementary tools and collectors for monitoring the many layers of a Kubernetes cluster,
```
Comment (OdysLam) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:17]: As a reader, I am not sure what you refer to here. Perhaps a bullet point list of the tools you refer to + links to their reference docs?

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 4]
```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
...
-Kubernetes cluster. Let's get started.
-
-## Prerequisites
-
-To follow this guide, you need:
-
--   A working cluster running Kubernetes v1.9 or newer.
--   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool, within [one minor version
+In this tutorial, you'll learn how to navigate Netdata's Kubernetes monitoring features, using
+[robot-shop](https://github.com/instana/robot-shop) as an example deployment. Deploying robot-shop is purely optional.
```
Comment (OdysLam) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:37]: It's awesome that we are using a real-life example. Let's try to put into our workflow the ideation of project examples that have a co-marketing/co-community value as well (e.g K6: addition of netdata in their docs, etc). I am thinking to play with Netdata + OpenFaaS which is deployed on K8s.

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 5]
```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
...
+robot-shop    rabbitmq-5bb66bb6c9-6xr5b         1/1     Running   0          14m
+robot-shop    ratings-94fd9c75b-42wvh           1/1     Running   0          14m
+robot-shop    redis-0                           0/1     Pending   0          14m
+robot-shop    shipping-7d69cb88b-w7hpj          1/1     Running   0          14m
+robot-shop    user-79c445b44b-hwnm9             1/1     Running   0          14m
+robot-shop    web-8bb887476-lkcjx               1/1     Running   0          14m
+```
 
-Remember that the parent pod is responsible for storing metrics from all the child pods and sending alarms.
+## Explore Netdata's Kubernetes monitoring charts
```
Comment (OdysLam) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:97]: Users are used to seeing the agent dashboard streamed over to the cloud. Perhaps it's worth to add a sentence explicitly stating that the new viz is only available to the cloud dashboard overview tab.

File: docs/guides/monitor/kubernetes-k8s-netdata.md [hunk 6]
```diff
@@ -18,261 +31,223 @@ customization, or integration with your preferred alerting methods.
...
+for complete customization. For example, grouping the top chart by `k8s_container_name` reveals new information.
 
-This drop might signal a fault or crash in a particular Kubernetes service or deployment (see `kubectl get services` or
-`kubectl get deployments` for more details). If the number of pods increases, it may be because of something more
-benign, like another member of your team scaling up a service with `kubectl scale`.
+![Changing time-series charts](https://user-images.githubusercontent.com/1153921/109075212-139b4380-76b6-11eb-836f-939482ae55fc.png)
 
-You can also view charts for the Kubelet API server, the volume of runtime/Docker operations by type,
-configuration-related errors, and the actual vs. desired numbers of volumes, plus a lot more.
+## Service metrics
```
Comment (OdysLam) [inline docs/guides/monitor/kubernetes-k8s-netdata.md:180]: Applications metrics?

Comment (OdysLam) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:180]: Moreover, I remember Manos saying that we are thinking to change "service" to "application" as it's more intuitive to users (relevant to alarm/alerts discussion).

**Review by OdysLam — DISMISSED** [review]

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 6 above)
Comment (ilyam8) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:180]: [Service](https://kubernetes.io/docs/concepts/services-networking/service/) has a special meaning in k8s, so sounds reasonable.

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 4 above)
Comment (joelhans) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:38]: With a bit more time to validate and plan these things, that would be awesome.

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 5 above)
Comment (joelhans) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:98]: I feel like it's already fairly-well explained here. The deployment doc, which they would have already read through, also explains the location of the new features.

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 6 above)
Comment (joelhans) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:181]: We'll have to engage with Manos on that some more. I don't that we should name something like MySQL an _application_—it's a piece of software that implements an application.

**Review by ilyam8 — APPROVED** [review]

**Review by OdysLam — APPROVED** [review]

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 4 above)
Comment (OdysLam) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:38]: That's a very very very valid comment. I will make a point today in product-marketing sync to underline that an additional benefit to having more time is to scout for opportunities just like this one.

File: docs/guides/monitor/kubernetes-k8s-netdata.md (hunk 6 above)
Comment (OdysLam) [reply docs/guides/monitor/kubernetes-k8s-netdata.md:181]: Both arguments are fair. The important part is to be aligned.

**Review by OdysLam — APPROVED** [review]

**Pull request merged**